chirps.py

```diff
@@ -0,0 +1,43 @@
+import pickle
+import uuid
+from datetime import datetime
+from time import time
+from user import *
+from service_functions import *
+
+class Chirps:
+  def __init__(self, user_id, user_screen_name, message, storage_file='chirps.txt'):
+    self.timestamp = timestamp()
+    self.chirp_id = generate_uuid()
+    self.user_uuid = user_id
+    self.screen_name = user_screen_name
+    self.message = message
+    self.public = True
+    self.chirp_data = {
```

Comment:
@Jessicashinjo Come and sit with me ASAP. Need to discuss what this code is doing.